packages/metro-config/src/__tests__/loadConfig-test.js

```diff
@@ -35,7 +35,19 @@ describe('loadConfig', () => {
 
     const result = await loadConfig({});
 
-    expect(result).toMatchSnapshot();
+    (expect(result): $FlowFixMe).toMatchSnapshot({
```

Comment:
I didn't know how to fix this... toMatchSnapshot doesn't seem to accept nested property matchers.